R/redisConnect.R

```diff
@@ -0,0 +1,21 @@
+#' redisConnect
+#'
+#' @import proto dplyr
```

Comment:
Nope.  I breaks the entire design. It belongs into the constructor--every db accessor library or package on the planet does it that way as best as I can tell. So please remove the R function.  

If you're unsure how to add to it to the C++ (re-read the three pointers I gave above) then I can do it.  I would appreciate additional testing.  Our Redis instances are password-free ...